### DISCUSSION TITLE
Can Sequel parse mysql SQL queries?

### DISCUSSION THREAD

**Discussion by leenyburger** [jeremyevans/sequel#2260] [General]
I'm using Sequel to connect to third party databases and execute SQL. 

My app writes SQL with AI then de-parses the queries to enforce tenancy boundaries on each table using `pg_query` for Postgres databases. 

I now need to support mysql and I was wondering if there is a plugin for `sequel` that will break queries into their ASTs? Or if there's a third party gem anyone knows about that does this?

**Comment by jeremyevans** [comment]
It is explicitly part of Sequel's design that it does not attempt to parse SQL.  So you'll have to look for other tools to do what you want.

**Reply by leenyburger** [reply]
Okay, thanks!
